src/spatialdata_plot/pl/basic.py

```diff
@@ -183,8 +183,11 @@ def render_shapes(
             Key in :attr:`anndata.AnnData.layers` or `None` for :attr:`anndata.AnnData.X`.
         palette
             Palette for discrete annotations, see :class:`matplotlib.colors.Colormap`.
```

Comment:
L185 and L186 disagree, no?

Reply:
because of the :class:`matplotlib.colors.Colormap`?